doc/concepts.rst

```diff
@@ -1,7 +1,18 @@
-CCF Overview
-============
+Overview
+========
 
-.. note:: Find more technical details in the :ref:`Technical Overview section <design/index:Technical Overview>`.
+This page gives a broad overview of the fundamentals of CCF. This is where you should start if you're new to CCF.
```

Comment:
consider moving the definition of CCF (Confidential Consortium Framework) here

Reply:
The acronym is expanded in first section of this page, a few lines below.